server/core/src/main/java/io/github/axisangles/ktmath/Matrix3.kt

```diff
@@ -41,6 +45,16 @@ data class Matrix3(
 	val yRow get() = Vector3(xy, yy, zy)
 	val zRow get() = Vector3(xz, yz, zz)
 
+	operator fun component1(): Float = xx
+	operator fun component2(): Float = yx
+	operator fun component3(): Float = zx
+	operator fun component4(): Float = xy
+	operator fun component5(): Float = yy
+	operator fun component6(): Float = zy
+	operator fun component7(): Float = xz
+	operator fun component8(): Float = yz
+	operator fun component9(): Float = zz
```

Comment:
What are these for?

Reply:
its for destructuring, its used in some places
```kt
val (_,x,y) = Quaternion(1,0,0,0)
```

Reply:
It wasn't here before because data classes implement component destructuring by default